runtime/src/test/java/com/dylibso/chicory/runtime/OperandStackTest.java

```diff
@@ -0,0 +1,86 @@
+package com.dylibso.chicory.runtime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.dylibso.chicory.wasm.types.Value;
+import org.junit.jupiter.api.Test;
+
+public class OperandStackTest {
+
+    @Test
+    public void push() {
+        var stack = new OperandStack();
+
+        assertEquals(0, stack.size());
+
+        stack.push(Value.i32(5));
+        stack.push(Value.i32(10));
+
+        assertEquals(2, stack.size());
+    }
+
+    @Test
+    public void pushWithResizing() {
```

Comment:
taking a quick look at this test, It feels like it's testing `ArrayDeque` and not necessarily our code. I think we can keep it scoped to error the conditions for now or whatever is in our impl of OperandStack.